### DISCUSSION TITLE
How should I report related bugs?

### DISCUSSION THREAD

**Discussion by WeatherWonders** [NickeManarin/ScreenToGif#1162] [Q&A] [answered]
I have a list of twelve closely related issues concerning image resolution with the Windows zoom at 125%. Half of these are about the recorder and half about the crop tool.

I can't tell whether some of these issues are the same bug. Should I open one issue for all of them, open one issue for the recorder and one for the crop tool, or try to split them up further as best I can?

With these issues aside, STG is excellent.

**Comment by NickeManarin** [comment, marked as answer]
Hello, you can write as one bug ticket. There's no problem.

I'm currently working on a complete rewrite (v3), so those fixes may come to that version instead of a v2.37.X.

:)
